src/main/java/com/google/cloud/tools/gradle/appengine/standard/DevAppServerStartTask.java

```diff
@@ -41,13 +43,23 @@ public void setCloudSdkBuilderFactory(CloudSdkBuilderFactory cloudSdkBuilderFact
     this.cloudSdkBuilderFactory = cloudSdkBuilderFactory;
   }
 
+  public void setDevAppServerLoggingDir(File devAppServerLoggingDir) {
+    this.devAppServerLoggingDir = devAppServerLoggingDir;
+  }
+
+  @OutputDirectory
+  public File getDevAppServerLoggingDir() {
+    return devAppServerLoggingDir;
+  }
+
   /** Task entrypoint : start the dev appserver (non-blocking). */
   @TaskAction
   public void startAction() throws AppEngineException, IOException {
 
     // Add a listener to write to a file for non-blocking starts, this really only works
     // when the gradle daemon is running (which is default for newer versions of gradle)
-    File logFile = File.createTempFile("dev_appserver", ".out", getTemporaryDir());
+    File logFile =
+        new File(devAppServerLoggingDir, "dev_appserver" + System.currentTimeMillis() + ".out");
```

Comment:
This will continue to build up log files until the user deletes the logging directory. This could be costly in storage without them ever realizing it; what about rewriting the same file? Or better yet, append to the same file with a max line limit? The latter may be over-engineering for this now so I think the former is the best option.